drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -21,6 +21,9 @@ struct dev_data_hfxo {
 	onoff_notify_fn notify;
 	struct k_timer timer;
 	sys_snode_t hfxo_node;
+#if IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS)
```

Comment:
It's ok like it's now but usually `IS_ENABLED()` is used for runtime check. So  `#if defined()` would be cleaner.